plugins/flytekit-whylogs/setup.py

```diff
@@ -0,0 +1,40 @@
+from setuptools import setup
+
+PLUGIN_NAME = "whylogs"
+
+microlib_name = f"flytekitplugins-{PLUGIN_NAME}"
+
+plugin_requires = [
+    "protobuf>=3.15,<4.0.0"
+    "whylogs",
```

Comment:
unintended string concatenation caused by lack of comma separating the values in these 2 lines.